pkg/api/addons.go

```diff
@@ -167,7 +175,7 @@ func (cs *ContainerService) setAddonsConfig(isUpdate bool) {
 
 	defaultMetricsServerAddonsConfig := KubernetesAddon{
 		Name:    MetricsServerAddonName,
-		Enabled: k8sVersionMetricsServerAddonEnabled(o),
+		Enabled: to.BoolPtr(common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.9.0")),
```

Comment:
If we're trying to be consistent, this should have a `DefaultMetricsServerAddonEnabled` just like heapster has a default enabled const.

Reply:
or we could remove `DefaultHeapsterAddonEnabled` because it doesn't make much sense to have it, we're basically doing `true && Something` <==>  `Something`

Reply:
Well, it allows us to change that in one place and update all the things that consume it accordingly (including addons default implementation). So I'll add one for metrics server.